bin/tessel-2.js

```diff
@@ -7,6 +7,7 @@ var key = require('../lib/key');
 var init = require('../lib/init');
 var logs = require('../lib/logs');
 
+
```

Comment:
https://github.com/tessel/t2-cli/blob/master/.jscsrc

But I don't think there is a rule that says "no new empty lines"